tests/Provider/Doctrine/Issues/Issue174Test.php

```diff
@@ -26,6 +23,6 @@ public function testIssue174(): void
         $configuration = new Configuration(['table_prefix' => 'audit_', 'table_suffix' => '_suffix']);
         $manager = new SchemaManager($this->provider);
         $result = $manager->computeAuditTablename('schema.entity', $configuration);
-        self::assertSame('schema.audit_entity_suffix', $result);
+        self::assertSame('audit_schema.entity_suffix', $result);
```

Comment:
Wrong, prefix is a table prefix so it goes at the beginning of the table name, not before the schema name (`schema_name.[PREFIX]table_name[SUFFIX]`)
=> `'schema.audit_entity_suffix'` is expected